app-editors/liteide/autobuild/build

```diff
@@ -0,0 +1,32 @@
+# From https://gitlab.archlinux.org/archlinux/packaging/packages/liteide/
+
+abinfo "Building liteide..."
+pushd build
+./update_pkg.sh
+./build_linux.sh
+popd
+
+abinfo "Installing liteide..."
+install -d \
+    "$PKGDIR"/usr/lib/liteide \
+    "$PKGDIR"/usr/share/liteide \
+    "$PKGDIR"/usr/share/doc/liteide
+for binary in gotools gocode liteide
+do
+    install -Dm755 "$SRCDIR"/build/liteide/bin/$binary $PKGDIR/usr/bin/$binary
+done
+cp -r "$SRCDIR"/liteidex/deploy/* os_deploy/* "$PKGDIR"/usr/share/liteide
+cp -r "$SRCDIR"/liteidex/liteide/lib/liteide/* "$PKGDIR"/usr/lib/liteide
+chmod -x "$PKGDIR"/usr/lib/liteide/plugins/*
+install -Dm644 "$SRCDIR"/liteidex/LICENSE.LGPL \
+    "$PKGDIR"/usr/share/licenses/liteide/LICENSE
+install -Dm644 "$SRCDIR"/liteidex/LGPL_EXCEPTION.TXT \
+    "$PKGDIR"/usr/share/licenses/liteide/LGPL_EXCEPTION
+install -Dm644 "$SRCDIR"/liteidex/liteide.desktop \
+    "$PKGDIR"/usr/share/applications/liteide.desktop
+install -d "$PKGDIR"/usr/share/pixmaps
+ln -s /usr/share/liteide/welcome/images/liteide400.png \
+    "$PKGDIR"/usr/share/pixmaps/liteide.png
+
+abinfo "Cleaning up..."
+rm -rf "$PKGDIR"/usr/share/liteide/doc
```

Comment:
Why?

Reply:
It seems that archlinux put the license file in /usr/share/licenses/. I have convert them to /usr/share/doc/ all.